plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/SqlServerQueryRunner.java

```diff
@@ -46,6 +45,7 @@ public static QueryRunner createSqlServerQueryRunner(
             Iterable<TpchTable<?>> tables)
             throws Exception
     {
+        DistributedQueryRunner.setupLogging();
```

Comment:
Bootstrap log is helpful in SQL Server connector when connecting to the instance from other tool (e.g. DBeaver) during the development because the connector generates the random database name in `TestingSqlServer`.

Reply:
Do you think we could log this explicitly in the QueryRunner instead of relying on the Bootstrap logger? The latter includes lots of irrelevant entries.

Reply:
In case of SQLServer test container there's a constant password, but if it were random, you wouldn't get it from the Bootstral logger anyway, so explicit logging might be better.

Reply:
Something like this:
```
2022-01-20T03:49:50.549-0600	INFO	pool-4-thread-1	io.trino.testing.DistributedQueryRunner	Created catalog sqlserver (sqlserver) in 421.28ms, with properties: {connection-password=A_Str0ng_Required_Password, connection-url=jdbc:sqlserver://localhost:62774;database=database_3a61c50f7ce84546b06bb56e2c313219;lockTimeout=60000, connection-user=sa, allow-drop-table=true}
```